components/Themes/src/MDCExperimentalColorScheme.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+@interface MDCExperimentalColorScheme : NSObject <NSCoding>
+
+@property(nonatomic, strong) UIColor *primaryColor;
```

Comment:
I know this is TBD, but I'm curious what a full scheme will look like.  Are we trying to develop a structure that contains every color an app may want?
+
primaryDarkColor, primaryLightColor;
secondaryColor, secondaryDarkColor, secondaryLightColor
errorTextColor